src/decorators/Model.ts

```diff
@@ -1,13 +1,14 @@
 const caller = require('caller'); // eslint-disable-line @typescript-eslint/no-var-requires
 import * as path from 'path';
 import { ObjectType } from 'type-graphql';
-import { Entity } from 'typeorm';
+import { Entity, EntityOptions } from 'typeorm';
 
 import { ClassType } from '../core';
 import { getMetadataStorage } from '../metadata';
 import { ClassDecoratorFactory, composeClassDecorators, generatedFolderPath } from '../utils/';
 
-export function Model() {
+// Allow default TypeORM options to be used
+export function Model(entityOptions: EntityOptions = {}) {
```

Comment:
Will need to allow for TypeQraphQL options to be passed, too. Perhaps we could use the same signature as `CustomField`: https://github.com/goldcaddy77/warthog/blob/master/src/decorators/CustomField.ts#L22